src/App.js

```diff
@@ -55,20 +64,4 @@ function MenuLink({ label, to, activeWhenExact = false }) {
   );
 }
 
-function ListView() {
```

Comment:
I'm leaving this comment here because I can't add it to the line _54_ of this file 😅 
What do you think if you change the structure of the function `MenuLink` like this:

`const MenuLink = ({ label, to, activeWhen....  }) => {}`

This way, we can keep more consistency in the code. 😄 

And also, what do you think about changing the "_**to**_" param name? With something more related to what it actually does, maybe something like "redirectTo" or just "path"? It's just an idea. 🥰